### PR TITLE
Issue46 guidelines open source

### PR DESCRIPTION
**Pull request by mortenwh**
#46

### CODE REVIEW

File: source/write_docs.rst [hunk 1]
```diff
@@ -1,7 +1,57 @@
-Writing and compiling the docs
-""""""""""""""""""""""""""""""
+Writing documentation
+"""""""""""""""""""""
 
-We use sphinx to build the documentation. To build the documentation locally, the best is to create
+All code should be documented. Integration with readthedocs is an easy an effective way
```
Comment (epifanio) [inline source/write_docs.rst]: typo: ... is an easy **and** effective way ...

Comment (mortenwh) [reply source/write_docs.rst]: fixed in b984525a0444f512883030a031a977bd950bd908

File: source/definition_of_done.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Definition of Done
+"""""""""""""""""""
+
+The following requirements must be satisfied before a task can be considered done:
+
+* The solution is in agreement with the design
+* 100% test coverage
```
Comment (havardf) [inline source/definition_of_done.rst]: I do not have any experience with the requirement of 100% test coverage. To me its seems something that might cost more than we gain?  We could loosen up the language and say "good enough" testing, ca. 70-90%? 

I will of course accept it, but I guess I am just saying I have no idea how much work that will be to achieve.

Comment (mortenwh) [reply source/definition_of_done.rst]: I think 100% coverage of the unit tests is quite important. It requires some extra effort by everyone, but my experience is that it is extremely useful in the long run... This is basically also what we already agreed about in the definition-of-done.

Comment (havardf) [reply source/definition_of_done.rst]: Yes, thats true, I was just thinking this through once more. Anyway, seems like I am in minority on this, so I will just accept it!

**Comment by havardf** [comment]
Approved!

**Comment by mortenwh** [comment]
need to add list of acronyms, and a terminology overview!

**Comment by mortenwh** [comment]
> need to add list of acronyms, and a terminology overview!

I have added this as issue #49

**Pull request merged**